pkg/utils/release/controller.go

```diff
@@ -5,11 +5,18 @@ import (
 
 	kubeCl "github.com/redhat-appstudio/e2e-tests/pkg/apis/kubernetes"
 	gitopsv1alpha1 "github.com/redhat-appstudio/managed-gitops/appstudio-shared/apis/appstudio.redhat.com/v1alpha1"
-	"github.com/redhat-appstudio/release-service/api/v1alpha1"
+	appstudiov1alpha1 "github.com/redhat-appstudio/release-service/api/v1alpha1"
+	kcp "github.com/redhat-appstudio/release-service/kcp"
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	// appstudiov1alpha1 "github.com/redhat-appstudio/release-service/api/v1alpha1"
```

Comment:
Why are you adding all of these if they are just being commented out?

Reply:
Just for debug